docs/specs/ia-case-documents-api.json

```diff
@@ -0,0 +1,304 @@
+{
```

Comment:
this should be published automatically, not through a PR, see https://github.com/hmcts/reform-scan-notification-service/blob/master/.github/workflows/swagger.yml

Reply:
Removed ia-case-documents-api.json and ia-case-notifications-api.json